src/main/res/layout/activity_result.xml

```diff
@@ -50,4 +50,23 @@
         android:layout_below="@+id/feedBack"
         android:layout_centerHorizontal="true" />
 
```

Comment:
Can you also add a text view beside the wager field so that the user can know what they need to type.